daemon/k8s_watcher.go

```diff
@@ -288,7 +288,19 @@ func (d *Daemon) EnableK8sWatcher(reSyncPeriod time.Duration) error {
 				},
 			},
 		)
+		d.k8sResourceSyncWaitGroup.Add(1)
 		go policyController.Run(wait.NeverStop)
+		go func() {
+			completed := make(<-chan struct{})
```

Comment:
Wondering whether we should ever do anything with this channel. I didn't see a need for it at this time.